docker/pypi/wmagent-couchdb/Dockerfile

```diff
@@ -79,22 +56,14 @@ RUN ln -s ${COUCH_CONFIG_DIR}/local.ini /opt/couchdb/etc/local.d/
 ENV PATH="/opt/couchdb/bin:/usr/local/bin/:${PATH}"
 ENV CRYPTOGRAPHY_ALLOW_OPENSSL_102=true
 
-# RUN <<EOF cat >> /opt/couchdb/.bashrc
-RUN <<EOF cat >> /home/$USER/.bashrc
-
-alias lll="ls -lathr"
-alias ls="ls --color=auto"
-alias ll='ls -la --color=auto'
-
-alias manage=$COUCH_MANAGE_DIR/manage
-
-# set CouchDB docker specific bash prompt:
-export PS1="(CouchDB-$TAG) [\u@\h:\W]\$ "
+# Set command prompt for root
+RUN <<EOF cat >> /root/.bashrc
+export PS1="(CouchDB-$TAG) [\u@\h:\W]# "
 EOF
 
-RUN chown -R ${USER}:${GROUP} ${COUCH_ROOT_DIR}
```

Comment:
HI @amaltaro  the removal of this line here is the origin of the problem explained in https://github.com/dmwm/CMSKubernetes/pull/1469#issuecomment-2083139403 

Unfortunately we cannot afford to keep it.  I am thinking about a workaround here.

Reply:
hi @amaltaro 
And I just tested  a workaround and suggested a PR to your branch: 
https://github.com/amaltaro/CMSKubernetes/pull/1 

There is still one error during `couchapp` installation, but now at least the database  user and admin accounts are set correctly:
```
cmst1@vocms0260:wmagent-couchdb $ ./couchdb-docker-run.sh  -t 3.2.2 && docker logs -f couchdb 
Starting couchdb:3.2.2 docker container with user: cmst1:zh
626c2f8650c8cd2cb529abeb2a13cfb4940f4dd9606e388ead9ecefeeecde1b1
Running CouchDB container with user: cmst1 (ID: 31961) and group: zh (ID: 1399)
'/usr/local/local.ini' -> '/data/srv/couchdb/3.2.2/config/local.ini'
init_couchdb: Adding a new account: cmst1
Which couchdb: /opt/couchdb/bin/couchdb
  With configuration directory: /data/srv/couchdb/3.2.2/config
  With logdir: /data/srv/couchdb/3.2.2/logs
  nohup couchdb -couch_ini /data/srv/couchdb/3.2.2/config >> /data/srv/couchdb/3.2.2/logs/couch.log 2>&1 &
Couchapps not found. Installing from latest WMCore tag.
mkdir: cannot create directory '/data/srv/couchdb/3.2.2/install/stagingarea': Permission denied
/data/manage: line 403: pushd: /data/srv/couchdb/3.2.2/install/stagingarea/tmp: No such file or directory
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100  4824  100  4824    0     0  26360      0 --:--:-- --:--:-- --:--:-- 26360

Pulling couchapps version latest from Github...
2.3.2.tar.gz: Permission denied
Cannot write to '2.3.2.tar.gz' (Permission denied).
Error pulling couchapps version latest from Github
start sleeping....zzz
```
I think this one is also solvable in the same manner. I' ll create yet another commit for moving/searching  the  `/data/srv/couchdb/3.2.2/install/` directory to the right place.

Reply:
And I have it all fixed with my PR I'll  give details in the main thread and not in the comment here